### DISCUSSION TITLE
get_images and async get_images error checking

### DISCUSSION THREAD

**Discussion by Zaazu** [Integration-Automation/ReEdgeGPT#116] [General]
Hi,

I've been using `ImageGenAsync.get_images` and noticed the error checking is much different and less verbose compared to `ImageGen.get_images` is this just an oversight as to why the code is so different?

Cheers

**Comment by JE-Chen** [comment]
Not a oversight, they have different behavior.

**Comment by Zaazu** [comment]
Alright, and is there no way to make the error handling more detailed like in the non async version?

**Comment by JE-Chen** [comment]
Maybe this conversation should be a issue not a discussion.

**Comment by JE-Chen** [comment]
And error handling more detailed, how detailed?

**Comment by Zaazu** [comment]
It should at least be as detailed as the non async version, and just so we're clear this is what I'm referring to:

Normal
https://github.com/Integration-Automation/ReEdgeGPT/blob/fbf25f61bca19953c595ee96c525b7406ac7b56a/re_edge_gpt/image/image_genearation.py#L125-L140

vs Async
https://github.com/Integration-Automation/ReEdgeGPT/blob/fbf25f61bca19953c595ee96c525b7406ac7b56a/re_edge_gpt/image/image_genearation.py#L319-L322

I can open an issue if you like

**Comment by JE-Chen** [comment]
> I can open an issue if you like 
Sure please open issue
